airflow/api_connexion/openapi/v1.yaml

```diff
@@ -6013,6 +6015,34 @@ components:
       required: false
       description: Returns objects matched by the Run ID.
 
+    FilterTimestampGTE:
+      in: query
+      name: timestamp_gte
+      schema:
+        type: string
+        format: date-time
+      required: false
+      description: |
+        Returns objects greater or equal the specified date.
+
+        This can be combined with timestamp_lte parameter to receive only the selected period.
+
+        *New in version 2.11.0*
```

Comment:
There will not be `2.11.0` release.

This is where it gets controversial because it's a feature, and we don't have any feature release for 2.x anymore.

Can this be added to bridge release ? (I think we said no feature) or squeezed into next patch release `2.10.x` ? @ephraimbuddy what do you think ?

Reply:
I'd love for it to get reeased in the next patch if possible!